samples/google-tag-manager.html

```diff
@@ -12,7 +12,7 @@
 </head>
 <body ng-app="sample">
 	<!-- Google Tag Manager, replace GTM-XXXXX with your container ID -->
-<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-53SG4H"
+<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-XXXXX"
```

Comment:
Should the number of Xs reflect the number of characters in the ID? 6 Xs instead of 5?